docs/concepts/index.asciidoc

```diff
@@ -115,6 +115,14 @@ Following are some example KQL queries.  For more detailed examples, refer to <<
 | `machine.os:win*`
 |===
 
+[float]
+[[autocomplete-suggestions]]
+==== Suggestions for autocomplete
+
+Beginning in 7.14, Kibana uses the https://www.elastic.co/guide/en/elasticsearch/reference/current/search-terms-enum.html[*terms enum API*] for autocomplete. Kibana returns results faster, but suggestions are approximate, sorted alphabetically, and can be outside the selected time range, even if `autocomplete:useTimeFilter` is enabled (as the terms enum API applies time filtering on an index-level, rather than document-level).
+
+Previously, Kibana used the https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html[*terms aggregation API*], which is slower, but suggestions included all values that matched your query, and optionally, your time range, and were sorted by popularity. To revert to the using the terms aggregation API, go to https://www.elastic.co/guide/en/kibana/current/advanced-options.html[*Advanced Settings*], and set `autocomplete:valueSuggestionMethod` to `terms_agg`.
```

Comment:
```suggestion
Previously, {kib} used the {ref}/search-aggregations-bucket-terms-aggregation.html[terms aggregation API], which is slower, but suggestions included all values that matched your query, and optionally, your time range, and were sorted by popularity. To revert to using the terms aggregation API, go to <<advanced-options, Advanced Settings>>, and set `autocomplete:valueSuggestionMethod` to `terms_agg`.
```

Reply:
Note:  You'll need to add the `autocomplete:valueSuggestionMethod` setting to the [Advanced Settings](https://www.elastic.co/guide/en/kibana/current/advanced-options.html) doc.

Reply:
The suggested changes to the links should fix the doc build errors.